frontend/src/components/TemplateEditor/controls/__snapshots__/ControlPanelCheckbox.test.js.snap

```diff
@@ -14,7 +14,6 @@ exports[`ControlPanelCheckbox component renders as expected 1`] = `
       >
         <input
           aria-invalid="false"
-          aria-label="creation.app.name"
```

Comment:
Rather than allowing these aria-labels to be removed from the snapshot, please set up the tests so that we either get the real translations or we get the translation keys as the values.